docs/src/pages/guides/server-rendering/server-rendering.md

```diff
@@ -171,7 +171,7 @@ function Main() {
   React.useEffect(() => {
     const jssStyles = document.querySelector('#jss-server-side');
     if (jssStyles) {
-      jssStyles.parentElement.removeChild(jssStyles);
+      jssStyles.parentNode.removeChild(jssStyles);
```

Comment:
Could you explain why we need to use `parentNode` instead of `parentElement` here?